src/bot/__main__.py

```diff
@@ -1,33 +1,52 @@
-"""Main runner"""
+"""Main runner."""
 
 import asyncio
+from collections.abc import Callable
 
-import aiohttp
 import discord
+from aiohttp import ClientSession, ClientTimeout
 from discord.ext import commands
 
 from bot import constants
 from bot.bot import Bot
 from bot.log import setup_sentry
 
+from .dragonfly_services import DragonflyServices
+
 setup_sentry()
 
 intents = discord.Intents.default()
 intents.message_content = True
 
 
+def get_prefix(bot_: Bot, message_: discord.Message) -> Callable[[Bot, discord.Message], list[str]]:
```

Comment:
```suggestion
def get_prefix(bot: Bot, message: discord.Message) -> Callable[[Bot, discord.Message], list[str]]:
```
I think using simply `bot` and `message` here is fine, since it will shadow any variables of the same name in an outer scope.